modules/monitoring-applying-a-custom-configuration-to-alertmanager-for-user-defined-alert-routing.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="applying-a-custom-configuration-to-alertmanager-for-user-defined-alert-routing_{context}"]
+= Applying a custom configuration to Alertmanager for user-defined alert routing 
+
+If you have enabled a separate instance of Alertmanager dedicated to user-defined alert routing, you can overwrite the configuration for this instance of Alertmanager by editing the `alertmanager-user-workload` secret in the `openshift-user-workload-monitoring` namespace.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Print the currently active Alertmanager configuration into the file `alertmanager.yaml`:
```

Comment:
since we already have file named `alertmanager.yaml`, 
http://file.rdu.redhat.com/bburt/RHDEVDOCS-3918-alertmanager-config-for-user-defined-monitoring/monitoring/managing-alerts.html#applying-custom-alertmanager-configuration_managing-alerts
it's better to change to `user-workload-alertmanager.yaml` to distinguish